packages/gluestick/src/generator/templates/EntryWrapper.js

```diff
@@ -143,7 +143,9 @@ export default class EntryWrapper extends Component {
     } = this.props;
 
     return rootWrappers.reduce((prev, curr) => {
-      return curr(prev, rootWrappersOptions);
+      const plugin = curr.ref.plugin || curr;
```

Comment:
👍 I'm not crazy about `ref` since I don't think it's common enough to be self-documenting, but you're keeping consistency with other plugins which is much more important.